cli/central-client/src/main/java/org/ballerinalang/central/client/Utils.java

```diff
@@ -469,7 +518,7 @@ public void writeTo(BufferedSink sink) throws IOException {
             final long totalBytes = contentLength();
             long byteConverter;
             String unitName;
-    
+
             if (totalBytes < 1024 * 5) { // use bytes for progress bar if payload is less than 5 KB
```

Comment:
It would be better to move these numbers and `unitName` to constants. Otherwise readers can get confused.
eg.- BYTES_FOR_KB = 1024;
PROGRESS_BAR_BYTE_THRESHOLD = 5;

Reply:
Fix in https://github.com/ballerina-platform/ballerina-lang/pull/41490/commits/99341c281ba6f7557ae22d012736bb1c6f1ce33f